WordPress/src/main/java/org/wordpress/android/ui/photopicker/mediapicker/MediaPickerViewModel.kt

```diff
@@ -335,119 +275,25 @@ class MediaPickerViewModel @Inject constructor(
         }
     }
 
+    fun performEditAction() {
+        val uriList = selectedURIs()
+        _navigateToEdit.value = Event(uriList)
+    }
+
     fun clickOnLastTappedIcon() = clickIcon(lastTappedIcon!!)
 
-    fun clickIcon(icon: MediaPickerIcon) {
-        if (icon == ANDROID_CAPTURE_PHOTO || icon == ANDROID_CAPTURE_VIDEO || icon == WP_STORIES_CAPTURE) {
+    private fun clickIcon(icon: MediaPickerIcon) {
+        if (icon == WP_STORIES_CAPTURE) {
             if (!permissionsHandler.hasPermissionsToAccessPhotos()) {
                 _onPermissionsRequested.value = Event(PermissionsRequested.CAMERA)
                 lastTappedIcon = icon
                 return
             }
-        }
-        when (icon) {
-            ANDROID_CAPTURE_PHOTO -> trackSelectedOtherSourceEvents(
```

Comment:
Probably I'm just missing something here, but shouldn't we track these events somewhere else? (sorry in case it's a stupid point 🙃).

Reply:
no worries. These icons were removed from the UI and the enum so this when no longer makes sense 👍